### PR TITLE
Create a Global Pool with ip_address_space is 'ipv4' or 'IPV4' fail

### PR DESCRIPTION
**Pull request by Kandarimegha**
## Description
In this commit:
- Fix Create a Global Pool with ip_address_space is 'ipv4' or 'IPV4' fail

The Modified code normalizes the ip_address_space value from pool_details to ensure it follows a consistent format (IPv4). It retrieves the ip_address_space value, converts it to uppercase, and then checks if it matches "IPV4". If it does, the value is standardized to "IPv4". This approach allows for flexibility in input format (e.g., IPV4, ipv4, etc.). 

## Type of Change
- [ Yes] Bug fix
- [ No] New feature
- [ No] Breaking change
- [ No] Documentation update

## Checklist
- [ Yes] My code follows the style guidelines of this project
- [ Yes] I have performed a self-review of my own code
- [ Yes] I have commented my code, particularly in hard-to-understand areas
- [ Yes] I have made corresponding changes to the documentation
- [ Yes] My changes generate no new warnings
- [ Yes] I have added tests that prove my fix is effective or that my feature works
- [ Yes] New and existing unit tests pass locally with my changes
- [ No] Any dependent changes have been merged and published in downstream modules
- [ Yes] All the sanity checks have been completed and the sanity test cases have been executed

## Ansible Best Practices
- [ Yes] Tasks are idempotent (can be run multiple times without changing state)
- [ No] Variables and secrets are handled securely (e.g., using `ansible-vault` or environment variables)
- [ Yes] Playbooks are modular and reusable
- [ No] Handlers are used for actions that need to run on change

## Documentation
- [ Yes] All options and parameters are documented clearly.
- [ Yes] Examples are provided and tested.
- [ Yes] Notes and limitations are clearly stated.

### CODE REVIEW

**Review by madhansansel — COMMENTED** [review]
Can you check the Sanity errors?

File: plugins/modules/network_settings_workflow_manager.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 # Copyright (c) 2024, Cisco Systems
 # GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-"""Ansible module to perform operations on global pool, reserve pool and network in Cisco Catalyst Center."""
```
Comment (madhansansel) [inline plugins/modules/network_settings_workflow_manager.py:6]: Can you check like below?

---
"""
Ansible module to perform operations on global pool, reserve pool, and network in Cisco Catalyst Center.
"""

Comment (Kandarimegha) [reply plugins/modules/network_settings_workflow_manager.py:6]: checked this way.. getting error for this

Comment (madhansansel) [reply plugins/modules/network_settings_workflow_manager.py:6]: Can you check other sanity errors?

**Review by madhansansel — APPROVED** [review]

**Pull request merged**